src/status_im/transactions/screens/pending-transactions.cljs

```diff
@@ -0,0 +1,69 @@
+(ns status-im.transactions.screens.pending-transactions
+  (:require-macros [status-im.utils.views :refer [defview]])
+  (:require [re-frame.core :as rf]
+            [status-im.components.common.common :as common]
+            [status-im.components.react :as rn]
+            [status-im.components.sticky-button :as sticky-button]
+            [status-im.components.status-bar :as status-bar]
+            [status-im.components.toolbar-new.actions :as act]
+            [status-im.components.toolbar-new.view :as toolbar]
+            [status-im.transactions.views.list-item :as transactions-list-item]
+            [status-im.transactions.views.password-form :as password-form]
+            [status-im.transactions.styles.screens :as st]
+            [status-im.utils.listview :as lw]
+            [status-im.utils.platform :as platform]
+            [status-im.i18n :as i18n]))
+
+(defn toolbar-view [transactions]
+  [toolbar/toolbar
+   {:background-color st/transactions-toolbar-background
+    :nav-action       (act/close-white #(do (rf/dispatch [:deny-transactions])
+                                            (rf/dispatch [:navigate-back])))
+    :border-style     st/toolbar-border
+    :custom-content   [rn/view {:style st/toolbar-title-container}
+                       [rn/text {:style st/toolbar-title-text
+                                 :font :toolbar-title}
+                        (i18n/label :t/pending-transactions)]
+                       [rn/text {:style st/toolbar-title-count
+                                 :font :toolbar-title}
+                        (count transactions)]]}])
+
+(defn render-separator-fn [transactions-count]
```

Comment:
we have separator renderer  https://github.com/status-im/status-react/blob/epic/ui-refresh/src/status_im/components/renderers/renderers.cljs you can add renderer with a condition, will be helpful